### PR TITLE
Schedule homework

### PR DESCRIPTION
**Pull request by kondorski**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule-bonus
🏅 Granted 10 XP for @kondorski

**Comment by ziobron** [comment]
⏱ 3 XP extra for @kondorski for on-time delivery

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule
🏅 Granted 12 XP for @kondorski

**Comment by ziobron** [comment]
⏱ 3 XP extra for @kondorski for on-time delivery

File: homework/schedule/schedule.hpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <chrono>
+#include <string>
+#include <functional>
+#include <thread>
```
Comment (lukasz-kukulka) [inline homework/schedule/schedule.hpp:5]: Alphabetic would be better

Comment (kondorski) [reply homework/schedule/schedule.hpp]: Good point, thanks.

**Review by lukasz-kukulka — APPROVED** [review]

File: homework/schedule/schedule.hpp [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+#include <chrono>
+#include <functional>
+#include <string>
+#include <thread>
+
+
+template<typename T, typename ... Args>
+void schedule (T func, const std::chrono::seconds(duration), Args... args) {
+    std::this_thread::sleep_for(duration);
+    func(args...);
```
[the remark this answers is not in the document]
Comment (nauka-programowania-MA) [reply homework/schedule/schedule.hpp:11]: Or invoke them into the function using perfect forwarding.

std::invoke(f, std::forward<Args>(args)...);

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**